docs/copier.md

```diff
@@ -0,0 +1,93 @@
+# Using Python Copier
+
+## 1. Install Python
+
+First, ensure you have Python installed on your system so that you can use `pip`, Python's package installer, in the instructions that follow. You can download Python from the official website: [python.org](https://www.python.org/).
+
+## 2. Install Copier
+
+On **Windows**, open Command Prompt (search for `cmd` in the Start menu).
+
+On **MacOS** open Terminal. You can find it in the Launchpad or in Applications > Utilities.
+
+On **Linux** open your Terminal.
+
+Run the following command to install Copier using pip:
+
+   ```sh
+   pip install copier
+   ```
+
+## 3. Invoke Copier with the GitHub Repo
+
+Navigate into the directory where you want to create your new web experiment project and run the following command:
+
+   ```sh
+   copier github.com:gt-sse-center/AutoRA-experiment-server .
```

Comment:
Should this be `copier copy github.com:gt-sse-center/AutoRA-experiement-server .`? Also, this may need a comment to indicate that the repository username will change once the template is officially deployed.

Reply:
good catch! when you look at something too long you really need more eyes...